homeassistant/components/switcher_kis/cover.py

```diff
@@ -133,3 +152,23 @@ async def async_set_cover_position(self, **kwargs: Any) -> None:
     async def async_stop_cover(self, **kwargs: Any) -> None:
         """Stop the cover."""
         await self._async_call_api(API_STOP, self._cover_id)
+
+
+class SwitcherSingleCoverEntity(SwitcherBaseCoverEntity):
+    """Representation of a Switcher single cover entity."""
```

Comment:
Update method should be moved from the base class to here as it is not common for both classes:
https://github.com/home-assistant/core/blob/b84ce69ab4254f28c0a0f146eced8bb0e5198004/homeassistant/components/switcher_kis/cover.py#L104-L110

Reply:
It is done.